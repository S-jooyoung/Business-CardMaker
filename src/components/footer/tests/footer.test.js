// import dependencies
import React from "react";

// import custom-component
import Footer from "../footer";

// import snapshot-testing methods
import renderer from "react-test-renderer";

describe("Footer", () => {
  let footerComponent;

  beforeEach(() => {
    footerComponent = <Footer />;
  });

  it("render", () => {
    const component = renderer.create(footerComponent).toJSON();
    expect(component).toMatchSnapshot();
  });
});
