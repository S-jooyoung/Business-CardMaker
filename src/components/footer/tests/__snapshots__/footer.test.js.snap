// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer render 1`] = `
<footer
  className="footer"
>
  <p
    className="title"
  >
    Make your value
  </p>
</footer>
`;
